Steps4Impact/Login/LoginViewController.swift

```diff
@@ -97,7 +97,27 @@ extension LoginViewController: LoginButtonDelegate {
     guard let result = result else { return }
     if result.isCancelled { return }
 
-    AKFCausesService.createParticipant(fbid: Facebook.id)
+    AKFCausesService.createParticipant(fbid: Facebook.id) { (result) in
+      if result.isSuccess {
+        AKFCausesService.getParticipant(fbid: Facebook.id) { (result) in
+          guard let participant = Participant(json: result.response) else {
+            return
+          }
+          guard participant.event == nil else { return }
+
+          AKFCausesService.getEvents { (result) in
+            guard let events: [Event] = result.response?.arrayValue?.compactMap({ (json) -> Event? in
+              Event(json: json)
+            }), events.count > 0 else { return }
+
+            AKFCausesService.joinEvent(fbid: Facebook.id, eventID: events[0].id!)
```

Comment:
no need to force unwrap, just get the `.first` element of the array and then chain the id in the guard and use that instead